### ISSUE TITLE
Need annotated geoduck proteome

### ISSUE THREAD

**Issue by laurahspencer** [sr320/LabDocs#528]
To try to expedite the Pecan run we could cut down the peptide list/database. Need an annotated proteome for this. 

The undigested background proteome with a trimmed header is located on the srlab working director in the [/Laura/DNR_geoduck](http://owl.fish.washington.edu/halfshell/index.php?dir=working-directory%2F17-03-07c%2FLaura%2FDNR_geoduck%2F) folder. 

Here's a direct link to the file: [Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.pep](http://owl.fish.washington.edu/halfshell/working-directory/17-03-07c/Laura/DNR_geoduck/Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.pep)

**Comment by sr320** [comment]
tldr : https://raw.githubusercontent.com/sr320/nb-2017/master/P_generosa/analyses/Geoduck-Proteome-stress.fa

> Fasta file of 615 proteins related to stress response.

---





```python
!head P_generosa/data/Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.pep
```

    >cds.comp100047_c0_seq1|m.5980
    NAECRDLYKIFTQILSVRSQEGKIVIPDEFATKIRNWLGNKEELFKEAHNQKIITFYNEY
    TREENTFNPIRGKRPMSVPDMPERKYIDQLSRKTQSQCDFCKYKTFTAEDTFGRIDSNFS
    CSASNAFKLDHWHALFLLKTH
    >cds.comp100068_c0_seq1|m.5981
    LFLDKSGKRICSFNNLTAVIEKATERASRIRLAKGLSQPKYLSCGNVDKVPAPGYLTASF
    TQLSVNKTRKDKGRNHLLLWDQTSSYSYIGPGIHYKDGKIRVNTT
    >cds.comp100097_c0_seq1|m.5982
    GTENLRICLKVIETYLLLGPREFLELYSGDLVHSLSNLLSDLRTEGVLLVLRVIELVLKS
    FPTEGPALFKSMLPEFLRAVLNKDEHPVVMSLYLTLFGRIVLQNQEFFWNFLDQMAMESH




```python
!perl -e '$count=0; $len=0; while(<>) {s/\r?\n//; s/\t/ /g; if (s/^>//) { if ($. != 1) {print "\n"} s/ |$/\t/; $count++; $_ .= "\t";} else {s/ //g; $len += length($_)} print $_;} print "\n"; warn "\nConverted $count FASTA records in $. lines to tabular format\nTotal sequence length: $len\n\n";' \
P_generosa/data/Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.pep \
> P_generosa/analyses/Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.tab

```

    
    Converted 35951 FASTA records in 249671 lines to tabular format
    Total sequence length: 11803569
    



```python
!head P_generosa/analyses/Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.tab
```

    cds.comp100047_c0_seq1|m.5980		NAECRDLYKIFTQILSVRSQEGKIVIPDEFATKIRNWLGNKEELFKEAHNQKIITFYNEYTREENTFNPIRGKRPMSVPDMPERKYIDQLSRKTQSQCDFCKYKTFTAEDTFGRIDSNFSCSASNAFKLDHWHALFLLKTH
    cds.comp100068_c0_seq1|m.5981		LFLDKSGKRICSFNNLTAVIEKATERASRIRLAKGLSQPKYLSCGNVDKVPAPGYLTASFTQLSVNKTRKDKGRNHLLLWDQTSSYSYIGPGIHYKDGKIRVNTT
    cds.comp100097_c0_seq1|m.5982		GTENLRICLKVIETYLLLGPREFLELYSGDLVHSLSNLLSDLRTEGVLLVLRVIELVLKSFPTEGPALFKSMLPEFLRAVLNKDEHPVVMSLYLTLFGRIVLQNQEFFWNFLDQMAMESHTDSSTILKSMLAAWVESIDSMTQPERRKLSALALASLLTVNYSVILESFGSIVSVCVQVLHDVCRVPVDEETAIQMDALVIRDGDNLGEDEHETEHEKRKRALTRK
    cds.comp100100_c0_seq1|m.5983		VEPINRQATDGVKKKKKKKTKLSDGVLPSNVKANSFTKFSQAPDNFTGSKLSSGSACSSGSCTAGLMLKVGEFCQSIRTSSSETNNGASEHSNSEQCREGAKSDAAIQGIGLELTRKPPIPAKAGKPSNNGATISPDILRHTDKSGTKGHKRQKSLPKQHYPMVEAIGASDFRKNGTCSHDNPRLNSDPSILTTAPNKWSLKKSNSAPSPSNSKQTLPQCGA
    cds.comp100100_c0_seq2|m.5984		VEPINRQATDGVKKKKKKKTKLSDGVLPSNVKANSFTKFSQAPDNFTGSKLSSGSACSSGSCTAGLMLKVGEFCQSIRTSSSETNNGASEHSNSEQCREGAKSDAAIQGIGLELTRKPPIPAKAGKPSNNGATISPDILRHTDKSGTKGHKRQKSLPKQHYPMVEAIGASDFRKNGTCSHDNPRLNSDPSILTTAPNKWSLKKSNSAPSPSNSKQTLSQSGKISGSQPPIFNSVNKPLNTGTTSPGGRNKKLPQCGVQALFYGSEDQFGANQTGGNKIIRKEGFAKGLNSHITNGLDPSKYTLDPSKCGGLLSNEGGSVMDNSHSLIITRNSPGAVNGTCNINGEHGLHDDCSDTKSDEGTVGGINIVEGSNPETKQIGTWKKIFGLIK*
    cds.comp100104_c2_seq1|m.5985		PALCVFNNSCFTKADIEGIHNLGIGSKGDDPTKTGQYGVGFNAVYHLTDTPSFLTKGPEVEKGETLCIFDPLCKNIPKATEVEPGIRCDVGVLKQNFPDVLLGYLEKEGLFDSGKGTLFRFPLRCKKSDISSEIITKEKM
    cds.comp100104_c3_seq1|m.5987		EKMDKILSEFTEIMFEALLFVKNVNCIRISNMTDEGIIEEYSVHASMSLEDKKNRQIFFNEIHAIIKENTNDKAAILKEGKKEIGYEMVIEDSNQRLTEWYIVQIFGFENSSDEITSAFRDKMLGQSPLGGVAVPLKKVSVSTEQFREKDFRHDSRTEGKYDGSKQHDASHEITKNNSTVNGNEFCGKAFCFLPLPIKTNLPVHVHGHFVLDHEARQHLGGGEPGYKQTWNESILSNIVANCWISAIKYIQTSLVLANLQNVSCEKAMVCLMENYHGHFPLLRNGSEGMDHYWTYLVRSFYNTVVKNHHALFPVVRFSEFKKSDFDKGTLERKFEHELTWVAIGQNSLQGVFNMIKKHVPLTGSLYHNFYVHFSENDARELESI
    cds.comp100104_c4_seq1|m.5988		MEDGTEYIDYKEQHSSIVAPDINSILRPTYELVIDDTDYETSSTMNILHGRILPDVAIEFGVKPKKIKCLDELSCAIPFGQKEKLVTRLQRLLDGYPCDSSIMKELLQNADDANASEMHFIKDYRTHSTEKIFGK
    cds.comp100105_c1_seq1|m.5989		ASQPYTENKCSFQISDSIQALSDVSPPGYEDNSDYQSLEGVHHGYYGDQSPTEFYPMIPEQKYRPPPIKRMCSGGFIPQEAHTQAQTESYFDQQHQYQMVPNIKTEMPWSQHEFPGCGRDLSGSWSPTDLRTNMSRVGYHPLHNLGEMGTQGPENKPMIQAAALAGYSGSGPIQLWQFLLEQLTDKSCQHFISWTGDGWEFKLADPDEVARRWGMRKNKPKMNYEKLSRGLRYYYDKNIIHKTAGKRYVYRFVCDLQSLLGYTPEELWEACDIKPQKDKDDE*
    cds.comp100108_c0_seq1|m.5990		AVTGIEADMWLSAISFLKNPTKDMLKEVKPLINMKDTTDKALLSVGTLIHSYCLRHGNCERDDMVKAVISAMQNKIAKGCKIKPDNFKTSLLALRGIGNAGYAVSAIPTLEICVKITSNPIEIRLSAIEAFRRMPCDASR



```python
!cat P_generosa/analyses/Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.tab | sed 's/cds.c/c/g' \
| tr '|' "\t" | sort > P_generosa/analyses/Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.sorted

```


```python
!head P_generosa/analyses/Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.sorted
```

    comp100047_c0_seq1	m.5980		NAECRDLYKIFTQILSVRSQEGKIVIPDEFATKIRNWLGNKEELFKEAHNQKIITFYNEYTREENTFNPIRGKRPMSVPDMPERKYIDQLSRKTQSQCDFCKYKTFTAEDTFGRIDSNFSCSASNAFKLDHWHALFLLKTH
    comp100068_c0_seq1	m.5981		LFLDKSGKRICSFNNLTAVIEKATERASRIRLAKGLSQPKYLSCGNVDKVPAPGYLTASFTQLSVNKTRKDKGRNHLLLWDQTSSYSYIGPGIHYKDGKIRVNTT
    comp100097_c0_seq1	m.5982		GTENLRICLKVIETYLLLGPREFLELYSGDLVHSLSNLLSDLRTEGVLLVLRVIELVLKSFPTEGPALFKSMLPEFLRAVLNKDEHPVVMSLYLTLFGRIVLQNQEFFWNFLDQMAMESHTDSSTILKSMLAAWVESIDSMTQPERRKLSALALASLLTVNYSVILESFGSIVSVCVQVLHDVCRVPVDEETAIQMDALVIRDGDNLGEDEHETEHEKRKRALTRK
    comp100100_c0_seq1	m.5983		VEPINRQATDGVKKKKKKKTKLSDGVLPSNVKANSFTKFSQAPDNFTGSKLSSGSACSSGSCTAGLMLKVGEFCQSIRTSSSETNNGASEHSNSEQCREGAKSDAAIQGIGLELTRKPPIPAKAGKPSNNGATISPDILRHTDKSGTKGHKRQKSLPKQHYPMVEAIGASDFRKNGTCSHDNPRLNSDPSILTTAPNKWSLKKSNSAPSPSNSKQTLPQCGA
    comp100100_c0_seq2	m.5984		VEPINRQATDGVKKKKKKKTKLSDGVLPSNVKANSFTKFSQAPDNFTGSKLSSGSACSSGSCTAGLMLKVGEFCQSIRTSSSETNNGASEHSNSEQCREGAKSDAAIQGIGLELTRKPPIPAKAGKPSNNGATISPDILRHTDKSGTKGHKRQKSLPKQHYPMVEAIGASDFRKNGTCSHDNPRLNSDPSILTTAPNKWSLKKSNSAPSPSNSKQTLSQSGKISGSQPPIFNSVNKPLNTGTTSPGGRNKKLPQCGVQALFYGSEDQFGANQTGGNKIIRKEGFAKGLNSHITNGLDPSKYTLDPSKCGGLLSNEGGSVMDNSHSLIITRNSPGAVNGTCNINGEHGLHDDCSDTKSDEGTVGGINIVEGSNPETKQIGTWKKIFGLIK*
    comp100104_c2_seq1	m.5985		PALCVFNNSCFTKADIEGIHNLGIGSKGDDPTKTGQYGVGFNAVYHLTDTPSFLTKGPEVEKGETLCIFDPLCKNIPKATEVEPGIRCDVGVLKQNFPDVLLGYLEKEGLFDSGKGTLFRFPLRCKKSDISSEIITKEKM
    comp100104_c3_seq1	m.5987		EKMDKILSEFTEIMFEALLFVKNVNCIRISNMTDEGIIEEYSVHASMSLEDKKNRQIFFNEIHAIIKENTNDKAAILKEGKKEIGYEMVIEDSNQRLTEWYIVQIFGFENSSDEITSAFRDKMLGQSPLGGVAVPLKKVSVSTEQFREKDFRHDSRTEGKYDGSKQHDASHEITKNNSTVNGNEFCGKAFCFLPLPIKTNLPVHVHGHFVLDHEARQHLGGGEPGYKQTWNESILSNIVANCWISAIKYIQTSLVLANLQNVSCEKAMVCLMENYHGHFPLLRNGSEGMDHYWTYLVRSFYNTVVKNHHALFPVVRFSEFKKSDFDKGTLERKFEHELTWVAIGQNSLQGVFNMIKKHVPLTGSLYHNFYVHFSENDARELESI
    comp100104_c4_seq1	m.5988		MEDGTEYIDYKEQHSSIVAPDINSILRPTYELVIDDTDYETSSTMNILHGRILPDVAIEFGVKPKKIKCLDELSCAIPFGQKEKLVTRLQRLLDGYPCDSSIMKELLQNADDANASEMHFIKDYRTHSTEKIFGK
    comp100105_c1_seq1	m.5989		ASQPYTENKCSFQISDSIQALSDVSPPGYEDNSDYQSLEGVHHGYYGDQSPTEFYPMIPEQKYRPPPIKRMCSGGFIPQEAHTQAQTESYFDQQHQYQMVPNIKTEMPWSQHEFPGCGRDLSGSWSPTDLRTNMSRVGYHPLHNLGEMGTQGPENKPMIQAAALAGYSGSGPIQLWQFLLEQLTDKSCQHFISWTGDGWEFKLADPDEVARRWGMRKNKPKMNYEKLSRGLRYYYDKNIIHKTAGKRYVYRFVCDLQSLLGYTPEELWEACDIKPQKDKDDE*
    comp100108_c0_seq1	m.5990		AVTGIEADMWLSAISFLKNPTKDMLKEVKPLINMKDTTDKALLSVGTLIHSYCLRHGNCERDDMVKAVISAMQNKIAKGCKIKPDNFKTSLLALRGIGNAGYAVSAIPTLEICVKITSNPIEIRLSAIEAFRRMPCDASR



```python
!head /Users/sr320/git-repos/paper-pano-go/data-results/Geoduck-transcriptome-v2-GO-SlimUnique.csv
```

    Column1,Column2,Column3,Column4,Column5,Column6,Column7,Column8,Column9,Column10,Column11,Column12,Column13,Column14,SPID,GOID,GO_id,term,GOSlim_bin,aspect,contig
    comp95_c0_seq1,sp,Q8K358,PIGU_MOUSE,67.53,77,25,0,231,1,258,334,7.00E-32,119,Q8K358,GO:0006506,GO:0006506,GPI anchor biosynthetic process,protein metabolism,P,95
    comp146_c0_seq1,sp,P37137,LHX5_XENLA,79.31,58,12,0,175,2,4,61,4.00E-31,116,P37137,GO:0006351,GO:0006351,"""transcription, DNA-dependent""",RNA metabolism,P,146
    comp195_c0_seq1,sp,Q8HXQ0,SODC_MACMU,54.84,62,28,0,188,3,80,141,1.00E-14,68.9,Q8HXQ0,GO:0009408,GO:0009408,response to heat,stress response,P,195
    comp296_c0_seq1,sp,P59966,DNAB_MYCBO,67.16,67,22,0,201,1,72,138,1.00E-11,63.2,P59966,GO:0006260,GO:0006260,DNA replication,DNA metabolism,P,296
    comp434_c0_seq1,sp,Q07954,LRP1_HUMAN,38.46,78,44,3,34,267,4094,4167,8.00E-10,59.3,Q07954,GO:0008283,GO:0008283,cell proliferation,cell cycle and proliferation,P,434
    comp455_c1_seq1,sp,Q9Y2K2,SIK3_HUMAN,84.85,66,10,0,200,3,65,130,1.00E-32,124,Q9Y2K2,GO:0016310,GO:0016310,phosphorylation,other metabolic processes,P,455
    comp943_c0_seq1,sp,P15586,GNS_HUMAN,41.89,74,39,3,235,17,385,455,3.00E-08,53.5,P15586,GO:0008152,GO:0008152,metabolic process,other metabolic processes,P,943
    comp945_c0_seq1,sp,Q14517,FAT1_HUMAN,49.09,55,25,2,51,212,4017,4069,2.00E-10,60.8,Q14517,GO:0007155,GO:0007155,cell adhesion,cell adhesion,P,945
    comp1059_c0_seq1,sp,Q6A7L9,SYP_PROAC,100,76,0,0,228,1,158,233,5.00E-45,156,Q6A7L9,GO:0006433,GO:0006433,prolyl-tRNA aminoacylation,protein metabolism,P,1059



```python
!grep "stress response" /Users/sr320/git-repos/paper-pano-go/data-results/Geoduck-transcriptome-v2-GO-SlimUnique.csv \
| cut -d, -f1 | sort > /Users/sr320/git-repos/nb-2017/P_generosa/analyses/Geoduck-stress-response.sorted 
```


```python
!join P_generosa/analyses/Geoduck-stress-response.sorted \
P_generosa/analyses/Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.sorted | wc -l 
```

         615



```python
!join P_generosa/analyses/Geoduck-stress-response.sorted \
P_generosa/analyses/Geoduck-transcriptome-v2.transdecoder_TrimmedHeader.sorted \
> P_generosa/analyses/Geoduck-Proteome-stress.txt
```


```python
!head P_generosa/analyses/Geoduck-Proteome-stress.txt
```

    comp100581_c0_seq1 m.6058 EGFQAELEEMDNIEYEYETDEKGNRVVLGRGSYGVVYAARDKRTQVRVAVKEVPEKFHEEVQPLHEEIKLHSRLSHKNIVKYLGSLSEEGYFKIFMEQVPGGSLSQLLSSKWGPLLDSEPTIAFYTKQILDGLKYLHDNRIVHRDI
    comp100905_c0_seq1 m.6111 IAVAGATYGQGAGPILLDDLECTGSETNIVQCRNKGWYSHNCGHSEDVGVECNAAAIQYRLVNGSNNKNGRVEVKLAGVWGTVCDDRFDTKAAKVVCKALNLPYHNAVPMTAATYGQGTGKIWLDE
    comp101656_c1_seq1 m.6227 QNTGSGDIPPVHGFTEEEILSVQTDLVLWYNENKRDLPWRRNATNSDVNQRAYAVWVSEIMLQQTQVATVIDYYNKWMKKWPTLASLALATLEEVNEMWSGLGYYSRGRRLHEGAKKVVEAFQGEMPKDAECLLKELPGVGKYTAGAIASIAYNERTGLVDGNVI
    comp102518_c0_seq1 m.6381 AAGPMWLEFNKQKIPAQLIQTYLNCFNYANEKLWITSLAVPAISTGVYGAPLDVCVRSFLDAVLIFTHNRHTLPRLREIKMVNRDAECTVTSIVLIQTLLETELDTMTRDALISMKTISESSARGLNLPRGRQEKNDFSKTL
    comp103523_c0_seq2 m.6531 MPPKRKATGGAAKGGKKAKKAEPAVPATMKDAAAVLKAADKKTGGKKSHKVDSNCPFASTATVEGDYDCMLNQTNIGHNNNKYYVIQLISCAGQYYAWNRWGRVGEKGATSLNGPSDLATAMKNFEKKFQDKTKNKWADRGKFIPKAGKYSLIEMADDEDEVDAPAPSVSTAGKKVAASELDKQTQNLVKLIFDHDMFKDCMKKFDIDVKKMPLGKLSKGQIAKGFEVLDEIEKELNANRKTKLNELSSRFYTVIPHNFGRTVPPCISTLEQLRGRMDMLLVLGDIEIAQSLQKEKDKKMENTKGEELPNALDVNYGLLNCKLDLVDKKSPEFKTIEMYTKNTSGGWRTPTIKEVWCVDRDGENSRF
    comp104030_c0_seq2 m.6605 IINTELTERLFSLANSIRQDLASLNIQRGRDHGLPFYNEYRKMINLTEAKSFDDLSNEIQDRQTIDNLELLYGHVDNIFVGGMAETPLEGAKVGPTFAYILADQFKRLREGDRFWYENPMVFSPDQLVQLKQASFARVICDSSDTLANVQRDVFTMVHSRDEYISCDDIPKIDLKTWSDCCMDCGKAGDFRSITSHFRRRRTTESSYQSDRPGTSSEQLQHFRQQVYNKHEMDTKMHEVENRLHEMEGLVHTME
    comp106352_c0_seq1 m.7009 PIPEKILGKMAVAIVNALHYLKENHGVIHRDVKPSNILLDERGMVKLCDFGISGRLVDSKAKTRSAGCAAYMAPERIDPPDPTRPDYDIRADVWSLGISMVEMATGEFPYKNCKTDFEVLTKVLQEDPPLLPAALNFSLDFCSFVKDCLTKDYRRRPKY
    comp106500_c0_seq1 m.7059 QGNEKLVNRLHHVKKLVKRYKRERKFLQGRLDTHNDNYKDAQVPVMWEEDQMYNLLKPPPLSPPTKELPDTPVSATTTAAAATTKPKSAKKIKSLLKSQETAVTGQSSGRPSSSIM
    comp107390_c0_seq1 m.7256 MSSEKRRRARVQGGWAPPTNVRSDRDKQKAQVTPNWFGKNVDQPAERKFVFEDSAEQLRQPPPSKVISKHGVFDISAEPSGLSRLRFFPMFLDAKEADMMHEALYHELPWKQRSDVKNGETYLQPRLTAWYGDHQYKYSGVMHEARTDWHPTLQTIKQRLEDNTGLQFNSMLANLYRDGKDNLAWHSDDETSLGPEPTIASISFGDTRTFQLRKKPPPEENGDYTYMQHIDVPLTHGSLLIMEGCTQADWQHRIPREYHDRSARINLTFRVIYEE*
    comp107463_c0_seq1 m.7274 EILENILKRKLYKCVGQTQLKHDQVIAKDKLSKVKEEIFNKMEVDELECSPLKRDDIIVHLAYLDYGMKDKNPIDNVRFYSKEDPNVAIRVRKIRYPRCYQKHSRNSSCEFTVRQEIRDLWT*



```python
!awk '{print ">"$1"\n"$3}' P_generosa/analyses/Geoduck-Proteome-stress.txt | head


```

    >comp100581_c0_seq1
    EGFQAELEEMDNIEYEYETDEKGNRVVLGRGSYGVVYAARDKRTQVRVAVKEVPEKFHEEVQPLHEEIKLHSRLSHKNIVKYLGSLSEEGYFKIFMEQVPGGSLSQLLSSKWGPLLDSEPTIAFYTKQILDGLKYLHDNRIVHRDI
    >comp100905_c0_seq1
    IAVAGATYGQGAGPILLDDLECTGSETNIVQCRNKGWYSHNCGHSEDVGVECNAAAIQYRLVNGSNNKNGRVEVKLAGVWGTVCDDRFDTKAAKVVCKALNLPYHNAVPMTAATYGQGTGKIWLDE
    >comp101656_c1_seq1
    QNTGSGDIPPVHGFTEEEILSVQTDLVLWYNENKRDLPWRRNATNSDVNQRAYAVWVSEIMLQQTQVATVIDYYNKWMKKWPTLASLALATLEEVNEMWSGLGYYSRGRRLHEGAKKVVEAFQGEMPKDAECLLKELPGVGKYTAGAIASIAYNERTGLVDGNVI
    >comp102518_c0_seq1
    AAGPMWLEFNKQKIPAQLIQTYLNCFNYANEKLWITSLAVPAISTGVYGAPLDVCVRSFLDAVLIFTHNRHTLPRLREIKMVNRDAECTVTSIVLIQTLLETELDTMTRDALISMKTISESSARGLNLPRGRQEKNDFSKTL
    >comp103523_c0_seq2
    MPPKRKATGGAAKGGKKAKKAEPAVPATMKDAAAVLKAADKKTGGKKSHKVDSNCPFASTATVEGDYDCMLNQTNIGHNNNKYYVIQLISCAGQYYAWNRWGRVGEKGATSLNGPSDLATAMKNFEKKFQDKTKNKWADRGKFIPKAGKYSLIEMADDEDEVDAPAPSVSTAGKKVAASELDKQTQNLVKLIFDHDMFKDCMKKFDIDVKKMPLGKLSKGQIAKGFEVLDEIEKELNANRKTKLNELSSRFYTVIPHNFGRTVPPCISTLEQLRGRMDMLLVLGDIEIAQSLQKEKDKKMENTKGEELPNALDVNYGLLNCKLDLVDKKSPEFKTIEMYTKNTSGGWRTPTIKEVWCVDRDGENSRF
    awk: write error on /dev/stdout
     input record number 119, file P_generosa/analyses/Geoduck-Proteome-stress.txt
     source line number 1



```python
!awk '{print ">"$1"\n"$3}' P_generosa/analyses/Geoduck-Proteome-stress.txt \
> P_generosa/analyses/Geoduck-Proteome-stress.fa

```


```python
!head P_generosa/analyses/Geoduck-Proteome-stress.fa
```

    >comp100581_c0_seq1
    EGFQAELEEMDNIEYEYETDEKGNRVVLGRGSYGVVYAARDKRTQVRVAVKEVPEKFHEEVQPLHEEIKLHSRLSHKNIVKYLGSLSEEGYFKIFMEQVPGGSLSQLLSSKWGPLLDSEPTIAFYTKQILDGLKYLHDNRIVHRDI
    >comp100905_c0_seq1
    IAVAGATYGQGAGPILLDDLECTGSETNIVQCRNKGWYSHNCGHSEDVGVECNAAAIQYRLVNGSNNKNGRVEVKLAGVWGTVCDDRFDTKAAKVVCKALNLPYHNAVPMTAATYGQGTGKIWLDE
    >comp101656_c1_seq1
    QNTGSGDIPPVHGFTEEEILSVQTDLVLWYNENKRDLPWRRNATNSDVNQRAYAVWVSEIMLQQTQVATVIDYYNKWMKKWPTLASLALATLEEVNEMWSGLGYYSRGRRLHEGAKKVVEAFQGEMPKDAECLLKELPGVGKYTAGAIASIAYNERTGLVDGNVI
    >comp102518_c0_seq1
    AAGPMWLEFNKQKIPAQLIQTYLNCFNYANEKLWITSLAVPAISTGVYGAPLDVCVRSFLDAVLIFTHNRHTLPRLREIKMVNRDAECTVTSIVLIQTLLETELDTMTRDALISMKTISESSARGLNLPRGRQEKNDFSKTL
    >comp103523_c0_seq2
    MPPKRKATGGAAKGGKKAKKAEPAVPATMKDAAAVLKAADKKTGGKKSHKVDSNCPFASTATVEGDYDCMLNQTNIGHNNNKYYVIQLISCAGQYYAWNRWGRVGEKGATSLNGPSDLATAMKNFEKKFQDKTKNKWADRGKFIPKAGKYSLIEMADDEDEVDAPAPSVSTAGKKVAASELDKQTQNLVKLIFDHDMFKDCMKKFDIDVKKMPLGKLSKGQIAKGFEVLDEIEKELNANRKTKLNELSSRFYTVIPHNFGRTVPPCISTLEQLRGRMDMLLVLGDIEIAQSLQKEKDKKMENTKGEELPNALDVNYGLLNCKLDLVDKKSPEFKTIEMYTKNTSGGWRTPTIKEVWCVDRDGENSRF



```python

```
